e2e_test/v2/streaming_1node/join-3way.slt

```diff
@@ -0,0 +1,50 @@
+statement ok
+SET RW_FORCE_DELTA_JOIN TO true;
+
+statement ok
+create table tt1 (v1 int, v2 int);
+
+statement ok
+create table tt2 (v3 int, v4 int);
+
+statement ok
+create table tt3 (v5 int, v6 int);
+
+statement ok
+create materialized view mv1 as select * from tt1, tt2, tt3 where tt1.v1 = tt2.v3 and tt1.v1 = tt3.v5;
+
+statement ok
+insert into tt1 values (0, 0), (1, 0), (2, 0), (3, 0);
+
+statement ok
+insert into tt2 values (0, 0), (1, 0), (2, 0), (3, 0);
+
+statement ok
+insert into tt3 values (0, 0), (1, 0), (2, 0), (3, 0);
+
+statement ok
+flush;
+
+query IIIIIII rowsort
+select v1, v3, v5 from mv1;
+----
+0 0 0
+1 1 1
+2 2 2
+3 3 3
+
+halt
+
+# TODO: drop MV is not supported yet for delta joins
```

Comment:
Any issue to track this?

Reply:
https://github.com/singularity-data/risingwave/issues/1901